addons/web/static/src/core/utils/draggable_hook_builder.js

```diff
@@ -775,6 +779,16 @@ export function makeDraggableHook(hookParams) {
              * @param {PointerEvent} ev
              */
             const updatePointerPosition = (ev) => {
+                if (ev.view === params.iframeWindow) {
+                    for (const iframe of document.getElementsByTagName("iframe")) {
+                        if (iframe.contentWindow === params.iframeWindow) {
+                            const boundingRect = iframe.getBoundingClientRect();
+                            ctx.pointer.x = ev.clientX + boundingRect.x;
+                            ctx.pointer.y = ev.clientY + boundingRect.y;
```

Comment:
The issue I see here is that if `boundingRect.y` is really big you will jump a lot lower than the Iframe (same with x). Is this thus intended? Wouldn't it make sense to only jump beyond the Iframe (using width and height).

Or you could set the pointer's x and y to either the iframe's x and y (top-left of iframe) or any other its boundaries?

Reply:
Normally (:crossed_fingers:) it should only be compensating

Basically the assumption here is that the drag and drop always uses the main view coordinates. But it subscribes to mouse movements inside the iframe if "iframeWindow" is provided, where the events won't actually have the right coordinates relative to the main view (they'll have the coordinates inside the iframe view, not the original view)

In the case where the drag and drop is actually used fully inside an iframe (which I think, now, is never), this code shouldn't do anything as it wouldn't be using "iframeWindow" and the drag-and-drop would only cover the inside of the iframe.

Maybe this diagram can help
![viewcoordinates](https://github.com/user-attachments/assets/cba98378-bac4-45a2-8ed9-0f6396f54b06)